InnerEye/ML/utils/metrics_util.py

```diff
@@ -63,22 +63,32 @@ def to_csv(self, file_name: Path) -> None:
         del sorted_by_dice[dice_numeric]
         sorted_by_dice.to_csv(file_name, index=False, float_format=self.float_format)
 
-    def save_aggregates_to_csv(self, file_path: Path) -> None:
+    def save_aggregates_to_csv(self, file_path: Path, allow_incomplete_labels: bool = False) -> None:
```

Comment:
I'm a little puzzled why we need this flag here. Shouldn't it be enough to just not write metrics to the dataframe, if there is not ground truth? Everything else should fall in place automatically then? At present, the code below is rather cryptic, and I personally don't get what is going on (comments would have helped too)

Reply:
When `allow_incomplete_labels` is false we leave the aggregates CSV file unchanged. When it is true we add a new column, `total_patients`, so that readers can easily compare the aggregated metric's patient count with the overall number of patients. That's why we need it. I'll make that more obvious in comments

Reply:
Done